arm-network/2016-09-01/swagger/routeFilter.json

```diff
@@ -0,0 +1,849 @@
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "routeFilterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the route filter."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of Route Filter Rule resources.",
+            "schema": {
+              "$ref": "#/definitions/RouteFilterRuleListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/serviceCommunities/{serviceName}": {
+      "get": {
+        "tags": [
+          "ServiceCommunities"
+        ],
+        "operationId": "ServiceCommunities_Get",
+        "description": "Get the available service communities.",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the service community."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of ExpressRouteServiceProdiver resources.",
+            "schema": {
+              "$ref": "#/definitions/ServiceCommunity"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/serviceCommunities": {
+      "get": {
+        "tags": [
+          "ServiceCommunities"
+        ],
+        "operationId": "ServiceCommunities_List",
+        "description": "Gets all the available service communities.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of ServiceCommunity resources.",
+            "schema": {
+              "$ref": "#/definitions/ServiceCommunityListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RouteFilterRulePropertiesFormat": {
+      "properties": {
+        "access": {
```

Comment:
Should the user specify the accessType while creating the rule? If it is not specified then what happens on the service side?